src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1129,6 +1123,8 @@ public enum PaxosOnLinearizabilityViolation
      */
     public ParameterizedClass default_compaction = null;
 
+    public volatile AccordSpec accord = new AccordSpec();
```

Comment:
don't need `volatile` here, `final` is better... we only mutate the leaf fields, not the complex fields